migrations/20221122001508_time_series__date__add_index.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX CONCURRENTLY time_series_date_idx ON time_series (date);
```

Comment:
This table has many rows in production (>600 million); `CREATE INDEX CONCURRENTLY` will avoid a lengthy table lock as this index is being created.